components/suggestion/at_mention_provider/at_mention_suggestion.jsx

```diff
@@ -112,12 +112,22 @@ export default class AtMentionSuggestion extends Suggestion {
         } else {
             itemname = item.username;
 
-            if ((item.first_name || item.last_name) && item.nickname) {
-                description = `${Utils.getFullName(item)} (${item.nickname})`;
-            } else if (item.nickname) {
-                description = `(${item.nickname})`;
-            } else if (item.first_name || item.last_name) {
-                description = `${Utils.getFullName(item)}`;
+            const fullName =
+                item.first_name || item.last_name ? `${Utils.getFullName(item)}` : '';
```

Comment:
The template strings for both of these aren't necessary. The full name one isn't needed at all, and the nickname one can have the brackets moved down to where we use the nickname

Reply:
✅  Updated as per request/discussion